endpointmanager/pkg/chplendpointquerier/chplendpointquerier.go

```diff
@@ -0,0 +1,20 @@
+package chplendpointquerier
+
+type EndpointList struct {
+	Endpoints []LanternEntry `json:"Endpoints"`
+}
+
+type LanternEntry struct {
+	URL              string `json:"URL"`
+	OrganizationName string `json:"OrganizationName"`
+	NPIID            string `json:"NPIID"`
+}
+
+func QueryCHPLEndpointList(chplURL string, fileToWriteTo string) {
+
+	if chplURL == "https://api.mhdi10xasayd.com/medhost-developer-composition/v1/fhir-base-urls.json" {
+		MedHostQuerier(chplURL, fileToWriteTo)
+	} else if chplURL == "https://nextgen.com/api/practice-search" {
```

Comment:
I'm thinking maybe it makes sense to make an enum like in `RuleOption` in `fhirendpointinfo.go` for the chplURLs. I think it's better to define at the top of the file `medHostURL = <url for medhost>` and then in the code `chplURL == medHostURL` from a clarity perspective, and then if the URL changes for whatever reason, or we end up needing to check the url multiple times throughout this function, then we don't have to keep writing the full URL out, which can be prone to errors. What do you think?